src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+/**
+ * Prune.
+ * .
+ * {code:bash}
+ * server prune months=6
+ * server prune months=6 --list
+ * server prune months=6 --force
+ * {code}
+ **/
+component {
+
+	// di
+	property name="serverService" inject="ServerService";
+
+	/**
+	 * Prune.
+	 *
+	 * @months.hint forget servers which last started date is greater or equal to months you set
+     * @list.hint give a list of servers which last started date is greater or equal to the months you set
+	 **/    
+    function run(
+        string months,
+        Boolean list        = false,   
+        Boolean force       = false,
+    ){
+        var serverInfo      = serverService.resolveServerDetails( arguments ).serverinfo;
+        var servers         = serverService.getServers();
+        var filterServers   = [];
+
+        print.line( "prune started...", 'yellow' ).toConsole();
+        if ( !arguments.force ){
+            servers.each( function( ID ){ runningServerCheck( servers[ arguments.ID ] ); } );
+        }
+
+        if( arguments.list )
+        {
+
+            generatePruneListServers( arguments.months, servers );
+        }
+        else
+        {
+
+            generatePruneListServers( arguments.months, servers );
+
+            var askContinuePrune = "Prune will forget and delete this servers! Do you still want to continue [y/n]?";
+
+            if( arguments.force || confirm( askContinuePrune ) ){
+
+                for( currentServer in servers ){
+                    result = dateDiff( "m", servers[ currentServer ].dateLastStarted, now() );
+                    if ( result >= arguments.months ){
+                        arrayAppend( filterServers, servers[ currentServer ] );
+                    }
+                }            
+
+                if ( arguments.force ){
+
+                    print.line( "using prune with force...", 'yellow' ).toConsole();
+
+                    var runningServers = getRunningServers( servers );
+                    /* areThereRunningServers = ( ! runningServers.isEmpty() ) */
+                    /* print.line( "are there running servers: #areThereRunningServers#" ).toConsole(); */
+
+                    if ( !runningServers.isEmpty() ) {
+
+                        var stopMessage = "Stopping server #serverInfo.name# first....";
+        
+                        print.line( stopMessage )
+                            .toConsole();
+        
+                        runningServers.each( function( ID ){
+                            var stopResults = serverService.stop( runningServers[ arguments.ID ] );
```

Comment:
The `server forget` command already has an option for it to stop the server first.  Why is this logic duplicated here as well.  Why not just call `server forget --stop` and let it handle everything?